honestcash-v2/src/app/modules/editor/services/editor.service.ts

```diff
@@ -41,6 +41,7 @@ export enum STORY_PROPERTIES {
   Title = 'title',
   Body = 'body',
   BodyJSON = 'bodyJSON',
+  BodyAndTitle = 'bodyAndTitle',
```

Comment:
We already have an endpoint for updating the title in the V1? Why did you decide to create a new one?

Reply:
user sees only one not saved indication meaning that it would be hard to manage code that has two different `NOT_SAVED` states and managing their API requests to save those properties. It would also make Not Saved to be present at higher rate because of title not being saved seperately.

I could have used forkJoin to do two different requests onSave but then if one fails there is no way to manage (with current code) the two different states of being `NOT_SAVED`.

We don't save hashtags on change on v2 only on click of publish button so actually there is not even the need to save these properties individually on the backend because we just save body and title on one batch then hashtags and the paid section properties on second batch so it would be better to revise it step 1 / step 2 manner but since we agreed I would do minimal change on backend I took the easy and least change requiring route with bodyAndTitle save.

Technically: EditorStoryPropertyChange works with Title, BodyJSON, Hashtags and not with BodyAndTitle. it is the EditorStoryPropertySave that uses BodyAndTitle to save post body and title in same batch. If you care about title being sent when body changes or vice versa (thus bloating request and increasing size) I can refine the code towards that aspect

Reply:
Yes. We would need to maybe determine what needs to be saved based on the body of the requests, not its path. I am merging it for now, but this is something that may be refactored in the future. @nynevi